spec.bs

```diff
@@ -1146,6 +1146,9 @@ A <dfn export>fenced frame config</dfn> is a [=struct=] with the following [=str
   
   : <dfn>is ad component</dfn>
   :: A [=boolean=]. Defaulting to false.
+
+  : <dfn>cross-origin reporting allowed</dfn>
+  :: A [=boolean=]. Defaulting to false.
```

Comment:
```suggestion
  :: A [=boolean=], initially false.
```

Just like the `[=fenced frame config instance/is ad component=]` dfn, which is what the `[=fenced frame config/is ad component=] definition should say too.